docs/audience.md

```diff
@@ -0,0 +1,133 @@
+---
+id: Audience
+title: Audience
+sidebar_label: Audience
+---
+
+Plant an App is a collaborative low-code development platform. Various roles from the team or from the stakeholder side contribute to building the application. Some will do simple changes, such as changing the body of an email that's sent by the system, while others will write code to extend the platform with new functionalities. But most of the application building happens somewhere in between these two extremes, through the low-code functionalities.
+
+This documentation is meant to cover all the roles that are typically involved in the development process. As you navigate through different sections, you will notice that each is marked with the appropriate audience that is intended for.
+
+As you go through each audience below, please keep in mind that this is not at all a clear separation of roles. It actually defines the minimum set of business and technical skills that would be needed to accomplish a task.
+
+## `Business Users`
+
+Business users usually come from the stakeholder side of a project. They have he ability to make micro-decisions regarding the application, such as what an email template should be, what fields should be captures in a form and so on. In a traditional development process, they would address their change requests to a business analyst that turns them into a specification for the developers. On Plant an App, they are able to do many changes by themselves, therefore saving a lot of back and forth.
+
+Skills: `decision making`, `business understanding`, `business tools savvy`
+
+Typical personas:
+
+- Project Manager on the stakeholder side
+- Business Analyst on the stakeholder side
+- Business Owner for SMBs
+- Department Lead
+- Digital Transformation Leader
+- External Digital Transformation Consultant
+
+Examples of activities:
+
+- Change the to, from, subject or body of an email generated by the system
+- Add or remove properties in the [Entities](/docs/Entities/Overview.md)
+- Adjust permissions for the users of the application
+- Build [Dashboards](/docs/Dashboards/Overview.md)
```

Comment:
Our files so far were organized as lower case names with '-' separator if needed.

Reply:
Let's discuss naming conventions in the next documentation meeting.